core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -45,6 +51,23 @@
     private Function<? super Throwable, ?> responseCauseSanitizer = DEFAULT_CAUSE_SANITIZER;
     private Function<? super HttpHeaders, ?> responseTrailersSanitizer = DEFAULT_HEADERS_SANITIZER;
 
+    /**
+     * Sets the {@link Logger} to use when logging.
+     * If unset, will use default logger that each logging implementation has.
+     */
+    public T logger(Logger logger) {
+        this.logger = requireNonNull(logger, "logger");
+        return self();
+    }
+
+    /**
+     * Returns the nullable {@link Logger} to use when logging.
+     */
+    @Nullable
+    public Logger logger() {
```

Comment:
Could be `protected` because we do not want to expose the getters in a builder.

Reply:
fixed